setup.py

```diff
@@ -38,7 +38,7 @@
         ],
     },
     install_requires=[
-        "dbt-core==1.0.1",
+        "dbt-core==1.6.2",
```

Comment:
Should this point to dbt-teradata (which pulls in dbt-core) instead?

Reply:
This is a dependency.
These test cases will run against the actual source code of dbt-teradata instead of installed dbt-teradata plugin

Reply:
ok. I will leave it as dbt-core==1.6.2

Reply:
Planning to release dbt-teradata 1.6.6
dbt-teradata 1.6.6 has a dependency with dbt-core 1.6.6